derive_builder_core/src/builder.rs

```diff
@@ -80,7 +83,7 @@ impl<'a> ToTokens for Builder<'a> {
                    impl_generics);
 
             tokens.append(quote!(
-                #[derive(Default, Clone)]
+                #[derive(Default, Clone #( , #derives)* )]
```

Comment:
I thought about this, but as long as these are going to be unconditionally present it felt strange to hide them far from the actual rendering of the builder.